docs/root/configuration/observability/access_log/usage.rst

```diff
@@ -516,6 +516,17 @@ The following command operators are supported:
   TCP/UDP
     Not implemented ("-")
 
+.. _config_access_log_format_downstream_transport_failure_reason:
+
+%DOWNSTREAM_TRANSPORT_FAILURE_REASON%
+  HTTP/TCP
+    If downstream connection failed due to transport socket (e.g. TLS handshake), provides the failure
```

Comment:
Do we generate access logs for failed TLS handshakes?

Reply:
Yes, one possible error for TLS termination is: `DOWNSTREAM_TRANSPORT_FAILURE_REASON=TLS_error:_268435709:SSL_routines:"
                           "OPENSSL_internal:NO_COMMON_SIGNATURE_ALGORITHMS`

Reply:
Note, that the DOWNSTREAM_TRANSPORT_FAILURE_REASON is valid only in the listener access log. The HTTP or TCP access logs would always observe empty DOWNSTREAM_TRANSPORT_FAILURE_REASON.

I think it needs to be explained in the doc to reduce possible confusion.

Reply:
Thanks! done!